Production/govuk_ios/Views/Home/Topics/TopicCard.swift

```diff
@@ -15,11 +15,8 @@ class TopicCard: UIView {
     }()
 
     private lazy var icon: UIImageView = {
-        let image = UIImage(systemName: viewModel.iconName)
-        let config = UIImage.SymbolConfiguration(pointSize: 24, weight: .semibold)
-        let imageView = UIImageView(image: image)
+        let imageView = UIImageView(image: viewModel.icon)
         imageView.tintColor = UIColor.govUK.fills.surfaceButtonPrimary
```

Comment:
I never noticed this until now, but this has a different tint colour from the onboarding, card. Can you check which one is correct, please?